### DISCUSSION TITLE
mcc

### DISCUSSION THREAD

**Discussion by ytfotuba22** [ORelio/Minecraft-Console-Client#2097] [Q&A]
how can i make it join a server from the hub

**Comment by milutinke** [comment]
Do you need to click on an item in an inventory or you can use commands?
If you need to use a command, you can follow this: https://github.com/MCCTeam/Minecraft-Console-Client/discussions/2091#discussioncomment-3392546

Just change this part:
```
wait 30
send /login yourpasswordhere
```

To:
```
wait 1
send /yourcommandhere
```

Example:
```
wait 1
send /skyblock
```

**Reply by milutinke** [reply]
If you need to click on an item, take a look at this: https://github.com/MCCTeam/Minecraft-Console-Client/discussions/2043
